src/libs/dutil/WixToolset.DUtil/butil.cpp

```diff
@@ -29,10 +30,13 @@ NOTE: caller is responsible for closing key
 ********************************************************************/
 static HRESULT OpenBundleKey(
     __in_z LPCWSTR wzBundleId,
-    __in BUNDLE_INSTALL_CONTEXT context, 
-    __inout HKEY *key);
-
+    __in BUNDLE_INSTALL_CONTEXT context,
+    __in_opt LPCWSTR szSubKey,
+    __inout HKEY* key);
 
+/********************************************************************
+BundleGetBundleInfo - Read the registration data for a gven bundle
+********************************************************************/
```

Comment:
FYI, these documentation comments are in the .cpp file in many places but they're supposed to be in the .h file so the consumers get them.